### ISSUE TITLE
True - false, істина - ???

### ISSUE THREAD

**Issue by AdriandeCita** [webdoky/content#28]
Потрібен якийсь відповідник для `false`. В тексті у вільному вигляді (маю на увазі ніяк не виділено форматуванням) зустрічаються обидва варіанти. `True` в тексті перекладається елементарно, а ось з `false` є проблема. А виділяти його кожен раз як фрагмент коду не хочеться (хоча й можна, звісно).

Поки я перекладав одну із статей, наштовхнувся на отаку дискусію, чим власне і скористався в перекладі.
https://ukrainian.stackexchange.com/questions/1409/%D0%AF%D0%BA-%D0%BF%D1%80%D0%B0%D0%B2%D0%B8%D0%BB%D1%8C%D0%BD%D0%BE-%D0%BF%D0%B5%D1%80%D0%B5%D0%BA%D0%BB%D0%B0%D1%81%D1%82%D0%B8-true-%D1%96-false-%D0%B2-%D0%B7%D0%BD%D0%B0%D1%87%D0%B5%D0%BD%D0%BD%D1%96-%D1%8F%D0%BA-%D1%82%D0%B5%D1%80%D0%BC%D1%96%D0%BD%D0%B8-%D0%B2-%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D1%83%D0%B2%D0%B0%D0%BD%D0%BD%D1%96

В дикій ~природі~ текстах зустрічається десь так:
```
    // 3. Нехай len буде значенням toUint32(lenValue).
    var len = O.length >>> 0;

    // 4. Якщо isCallable(callback) є хибою, викинемо виняток TypeError.
```

**Comment by IhorHordiichuk** [comment]
Я не проти «хибності». Але варто уточнювати, що «хибне твердження», оскільки хиба й вада надто загальні слова, які можуть стосуватися чого завгодно. Саме це я й мав на увазі. Бо сама «хиба» в цьому контексті сприймається, що хибно написаний вираз, а не булевий результат. З булевим теж треба визначитись. Вживаються і булевий, і логічний тип. Обидва — запозичені й рівнозначні. Головне послідовно вживати однаково, щоб не було мішанини.

**Comment by undead404** [comment]
Істина + хиба. На практиці часто вживають англіцизми. "Хиба" ж дуже рідко вживане слово в принципі, ніхто в реальних дискусіях не зве позицію опонента "хибою" уже, певне, багато років :-)

**Comment by undead404** [comment]
Microsoft Language Portal дає "хибність"

**Comment by AdriandeCita** [comment]
Отже, "істина" і "хибність"?

В процесі перекладу я помітив іще одну проблему: термін "хиба" має перетин із перекладом слова "bug" - ймовірно, саме це ти мав і на увазі @IhorHordiichuk. З цієї точки зору "хибу" краще заборонити взагалі.

**Comment by IhorHordiichuk** [comment]
Так, я мав на увазі, що може хибно тлумачитися у певних випадках. Але забороняти не варто, просто за контекстом слідкувати й не буде проблем. Правильно переданий контекст усуває вади перекладу.

**Comment by undead404** [comment]
Я б лишив "хибу" не рекомендованим варіантом, бо з тексту "name – хиба" тяжко було б зрозуміти, чи йдеться про помилку (ваду, баг), чи про те, що в змінній лежить значення `false`.
